manifests/sidecar-main_attacher_role.yaml

```diff
@@ -0,0 +1,16 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: openshift-csi-main-attacher-role
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Storage
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list", "watch"]
```

Comment:
I would expect some rules for `volumeattachment` - the attacher reads/writes them a lot

Reply:
Yes, that's right. The attacher will bind to openshift-csi-volumeattachment-reader-role and openshift-csi-volumeattachment-writer-role. I didn't include these rules into `openshift-csi-main-attacher-role` to avoid verb duplication.